examples/3-mutation/package.json

```diff
@@ -19,7 +19,7 @@
   "devDependencies": {
     "bs-platform": "5.0.4",
     "graphql_ppx": "^0.2.8",
-    "webpack": "^4.29.6",
+    "webpack": "4.40.2",
```

Comment:
Any reason for this particular change? I'd just as soon leave this out for now, unless it's absolutely critical to the example running.

Reply:
got an error when running so re-installed. Let me check it.

Reply:
back to `"webpack": "^4.29.6"`